app/layouts/default-layout.tsx

```diff
@@ -108,11 +108,11 @@ const DefaultLayout: React.FunctionComponent<Props> = ({
           .with-shadow {
             box-shadow: 1px 8px 13px -5px #00336694;
           }
-          .accordion button {
-            text-align: left;
-            padding-left: 0;
-            color: #1a5a96;
-          }
+          // .accordion button {
+          //   text-align: left;
+          //   padding-left: 0;
+          //   color: #1a5a96;
+          // }
```

Comment:
@dleard This CSS rule in the default layout was interfering with my accordion CSS, and I discovered that this rule seems unnecessary here anyway? I haven't been able to see if removing it has messed up anything else. It might just be defunct - Happo should highlight any differences.

Before merging, this commented CSS should just be removed (provided it doesn't mess with anything).